.pre-commit-config.yaml

```diff
@@ -8,42 +14,12 @@ repos:
         files: \.py$
         types: []
 
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
-    hooks:
-      - id: isort
-
-  - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.4
-    hooks:
-    - id: bandit
-      args: [--severity-level=high]
-      files: ^src/
```

Comment:
How does Ruff deal with limiting the Bandit severity and files?

Reply:
It doesn't, at present. The current configuration enables all Bandit rules except the three ignored ones, so in that sense it's stricter than before.

If Bandit should be disabled altogether for tests, that could be done with per-file-ignores.

Reply:
charliermarsh/ruff#3239